inc/knowbaseitem.class.php

```diff
@@ -573,12 +577,8 @@ static public function getVisibilityCriteria(bool $forceall = false): array {
          $where['OR'] = [
                'glpi_knowbaseitems.users_id'       => Session::getLoginUserID(),
                'glpi_knowbaseitems_users.users_id' => Session::getLoginUserID(),
+               'glpi_knowbaseitems.is_faq'         => 1,
```

Comment:
Are you sure of this change ?

Reply:
Currently, self-service users only have access to FAQ articles. The goal is to share with them, even if the article is not in the FAQ.